### ISSUE TITLE
ResNet50 and PAD-net

### ISSUE THREAD

**Issue by rongmq8802** [SimonVandenhende/Multi-Task-Learning-PyTorch#13]
First of all, thank you for opening up the code. I am learning multi-task learning recently. And from your paper "Multi-Task Learning for Dense Prediction Tasks: A Survey", it can be concluded that on NYDU-v2 dataset, the PAD-Net based on ResNet-50 seems to get the best results, but your program does not support this situation. May I ask if you can provide one or whether the current code can achieve this function.

**Comment by SimonVandenhende** [comment]
Hello, thanks for the interest. 

Indeed, the current code does not include PAD-Net using a ResNet-50 backbone. 
I took the number from the original paper, as the overall settings are very similar. 
From my personal experience, I found it difficult to reproduce the exact numbers from the PAD-Net paper.
It might be better to talk to the authors directly about this.
Note that I did spend some time tuning the HRNet-based version of PAD-Net, so that implementation should be fine I believe.
